src/Websockettest.js

```diff
@@ -259,7 +275,7 @@ const RMBTTest = (function() {
                 });
 
                 _intermediateResult.upBitPerSec = results.speed;
-                _intermediateResult.upBitPerSecLog = (log10(_intermediateResult.upBitPerSec / 1e6) + 2) / 4;
+                _intermediateResult.upBitPerSecLog = (Math.log10(_intermediateResult.upBitPerSec / 1e6) + 2) / 4;
```

Comment:
thanks :)
however, this is not supported by Internet Explorer (https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Math/log10). Please add a polyfill to the Helpers.js